contents/testing/integration-testing.md

```diff
@@ -0,0 +1,184 @@
+<frontmatter>
+  title: Integration Testing
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Integration Testing
+
+**Author(s): [Jacob Li PengCheng](https://github.com/jacoblipech)**
+
+Reviewers: [Amrut Prabhu](https://github.com/amrut-prabhu), [Marvin Chin Kun Song](https://github.com/marvinchin), [Lin Si Jie](https://github.com/sijie123)
+
+## What is integration testing?
+
+Integration testing is a part of software testing where individual parts of your application are combined and tested as a group. This usually happens after the [<tooltip content="Testing of individual module units within a system">unit testing stage</tooltip>](https://www.guru99.com/unit-testing-guide.html) and before the [<tooltip content="Evaluation of the software against requirements gathered from users and system specifications">system testing stage</tooltip>](https://www.guru99.com/system-testing.html).
+Integration testing combines individually tested modules as a group and aims to discover faults related to the interactions between these modules.
+
+Let's look at an example to better understand what these means.
+Suppose that you have built an online food ordering system and the architecture diagram of your application is shown as follows:
+
+<center>
+
+![Sample architecture diagram](integration-test/sample-architecture.png "Sample architecture diagram")
+</center>
+
+Zooming in on the food ordering aspect of this system, a unit test on the `Order Summary` checks for the correctness of the order details recorded and a unit test on the `Order Food` checks that the food ordered is in the correct format.
+
+After completion of unit testing on individual modules, an **integration test** on `Order Food` and `Order Summary` checks that upon ordering food, the order summary reflects the correct food orders.
```

Comment:
This section feels like a description of the different levels of testing. I think it would be better if you elaborate on the integration testing part a bit more.  

Can add stuff like this:
> In other words, integration testing checks whether the `Order Food` module _interacts_ correctly with the `Order Summary` module.